### PR TITLE
Views

### PR DESCRIPTION
**Pull request by smunoz1988**
Hello!

For this milestone, I made these changes:

- add functionality to page.
- use methods for deploy data (recent_posts and recent_comments)
- add data to show on deploy
- create helpers for users and posts
- implement a design based on the sneak peek wireframes.

users:

image

show user:

image

posts:

image

show post:

image

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,29 @@
...
+    <h4> Bio</h4>
+    <p><%= @user.bio %></p>
+</div>
+
+<%= render partial: 'shared/posts_container', locals: { posts: @user.posts } %>
+
+
+<div>
+  <div class="seeAllPostBtn">
+    <button >See all posts</button>
```
Comment (Owusu-Desmond) [inline app/views/users/show.html.erb]: - [ ] Please make sure that clicking on the **See all Post** link opens the user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png) :pray:  you can implement this like :point_down: 
  ```ruby
  <button>
  <%= link_to 'See all Post', user_posts_path(@user) %>
  </button>
  ```

File: app/views/shared/_posts_container.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="usersPosts">
+<% @user.recent_posts.each do |post| %> 
+    <div class="post">
+      <h4><%= link_to post.title, user_posts_path(user_id: post.author_id), class: 'custom-link-s' %></h4>
```
Comment (Owusu-Desmond) [inline app/views/shared/_posts_container.erb]: - [ ] I suggest that clinking on a post title links to the post-show page(`user_post_path`), not the posts index page (`user_posts_path`) :blush:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @smunoz1988 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/shared/_posts_container.erb (hunk 2 above)
Comment (smunoz1988) [reply app/views/shared/_posts_container.erb]: Great idea @Owusu-Desmond !! 

done!

File: app/views/users/show.html.erb (hunk 1 above)
Comment (smunoz1988) [reply app/views/users/show.html.erb]: @Owusu-Desmond  done!

thanks for the suggestion!

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @smunoz1988        , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  . 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] No linter errors
- [x] App Works as intended 💯
- [x] Views setup correctly


 

 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Pull request merged**